tools/build/make.py

```diff
@@ -205,13 +213,12 @@ def default_cross_toolchain():
         # if not os.getenv("X_COMPILER_TYPE"):
         #    new_env_vars["X_COMPILER_TYPE"] = parsed_args.cross_compiler_type
 
+    libbsd_install_dir = bootstrap_libbsd(source_root, objdir_prefix)
+    bmake_install_dir = bootstrap_bmake(source_root, objdir_prefix,
```

Comment:
bootstrap_bmake() could return the bmake_binary path directly, instead of the install directory.

Reply:
Yes that would make it a bit simpler further down, if you feel like updating this.